src/lib/__tests__/date.test.ts

```diff
@@ -1,51 +1,271 @@
 import { isNull } from "lodash"
 import moment, { Moment } from "moment"
-import { exhibitionPeriod, exhibitionStatus } from "lib/date"
+import {
+  dateRange,
+  exhibitionStatus,
+  datesWithHours,
+  datesWithDaysAndHours,
+  singleDateWithoutHour,
+  singleDate,
+  singleHour,
+  formattedOpeningHours,
+  datesAreSameDay,
+  formattedHours,
+} from "lib/date"
 
-describe("date", () => {
-  describe("exhibitionPeriod", () => {
-    it("includes the start and end date", () => {
-      const period = exhibitionPeriod(
-        moment("2011-01-01T00:00-0400"),
-        moment("2014-04-19T00:00-0400")
+describe("date formatting", () => {
+  describe("formattedHours", () => {
+    it("If within same am/pm", () => {
```

Comment:
This is not a proper sentence and doesn’t describe the behaviour, which is important for people to judge if the code is working as it should. E.g. something like:

```js
it("drops the period component if start and end time are within the same period", () => {
  // …
})
```

Reply:
(Same applies to all other tests.)